string checked_field_topic
string checked_field_name
string limit_type
float32 limit_value
